### PR TITLE
#6Task. sergey.vialov

### PR DESCRIPTION
**Pull request by yippiekay**
https://travis-ci.com/github/yippiekay/js-assignments/builds/180731920

### CODE REVIEW

**Comment by yippiekay** [comment]
If it necessary, link for last commit on travis
https://travis-ci.com/github/yippiekay/js-assignments/builds/175705350

**Comment by yippiekay** [comment]
If it is necessary, a link for the last commit on the travis
https://travis-ci.com/github/yippiekay/js-assignments/builds/175855018

File: task/06-conditions-n-loops-tasks.js [hunk 1]
```diff
@@ -166,7 +199,30 @@ function isInsideCircle(circle, point) {
...
+        let flag = false; 
+        
+        for(let letter = 0; letter < str.length - 1; letter++){
+            
+            flag = true;
+            
+            if(currentLetter == letter){
+                continue;
+            }
+            if(str[currentLetter ] == str[letter]){
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: "operator === is always preferable
sometimes ''==' may cause an error
(check at console :
"0" == 0
and "0" === 0)"

File: task/06-conditions-n-loops-tasks.js [hunk 2]
```diff
@@ -226,7 +286,15 @@ function reverseString(str) {
...
-    throw new Error('Not implemented');
+    
+    let result = 0;
+    
+    while(num){
+        result = num % 10 + result * 10;
+        num = Math.floor(num /= 10);
+    }
+
+    return result;
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: We can make the solution as in previous task (reverseString), but before we should convert number to string 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toString

File: task/06-conditions-n-loops-tasks.js [hunk 3]
```diff
@@ -296,7 +397,35 @@ function getDigitalRoot(num) {
...
+            if( popElem == '[' && elem == ']'){
+                continue;
+            }
+            if( popElem == '{' && elem == '}'){
+                continue;
+            }
+            if( popElem == '(' && elem == ')'){
+                continue;
+            }
+            if( popElem == '<' && elem == '>'){
```
Comment (vramaniuk) [inline task/06-conditions-n-loops-tasks.js]: operator === is always preferable

**Review by vramaniuk — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by vramaniuk** [comment]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**